hack/mkdocs-generate-conformance.py

```diff
@@ -41,8 +41,8 @@ def on_files(files, config, **kwargs):
     log.info("generating conformance")
 
     vers = getConformancePaths()
-    # Iterate over the list of versions. Exclude the pre 1.0 versions.
-    for v in vers[3:]:
+    # Iterate over the list of versions. Exclude the pre 1.1 versions.
```

Comment:
Hmm, this implies that v1.0 of Gateway API is no longer supported, which I don't know if we really want at the moment.

I think that it's probably better to let the versions stack up for now - since the conformance reports are staying around, it costs very little to leave implementations that only support v1.0 with links here, especially if something like @howardjohn's proposal in #3814 goes ahead and we need to start keeping track of "submitted a conformance report since v1.0" or something.

tl;dr I think we should leave this at v1.0 and above, and not remove the v1.0 page. For now. We can have a discussion about how many versions of Gateway API are supported another time, but let's not do it accidentally here.

Reply:
Agreed we probably need to adopt some set timeframe or number of versions for when to roll off old reports, and/or maybe alternatively consider a tweaked web UX so this is a single filterable page instead of separate pages.